<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AracFormu</class>
 <widget class="QWidget" name="AracFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1638</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Araç Bilgileri</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>56</y>
     <width>391</width>
     <height>593</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Araç Türü</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QCheckBox" name="chk_kullanim_disi">
      <property name="text">
       <string>KULLANIM DIŞI MI ?</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Yakıt Türü</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="cmb_kasa_tipi"/>
    </item>
    <item row="15" column="1">
     <widget class="QDoubleSpinBox" name="spin_gunluk_bedel">
      <property name="maximum">
       <double>999999999999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="cmb_vites"/>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="txt_sasi_no"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Kasa Tipi</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Model</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Üretim Yılı</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Kapı Sayısı</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="spin_kapi">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Vites</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Motor Gücü (HP)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txt_marka"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Renk</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="spin_motor_hacmi">
      <property name="maximum">
       <number>20000</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spin_motor_gucu">
      <property name="maximum">
       <number>20000</number>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Motor Hacmi (cc)</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Günlük Kiralama Bedeli (TL)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cmb_arac_turu"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Şasi No</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Marka</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="txt_motor_no"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Çekiş</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="txt_renk"/>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="cmb_cekis"/>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="cmb_yakit_turu"/>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spin_uretim_yili">
      <property name="minimum">
       <number>1900</number>
      </property>
      <property name="maximum">
       <number>2100</number>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Motor No</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txt_model"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbl_menzil">
      <property name="text">
       <string>Menzil (km):</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="txt_menzil"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_kaydet">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>720</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>KAYDET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_guncelle">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>720</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>GÜNCELLE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sil">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>720</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SİL</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>911</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>ARAÇ LİSTESİ</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="txt_arama"/>
    </item>
    <item row="5" column="0">
     <widget class="QTableWidget" name="tbl_liste"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>ARAMA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
